buildpipeline/pipeline.json

```diff
@@ -20,7 +20,7 @@
           "Parameters": {
             "PB_DistroRid": "rhel.7.2-x64",
             "PB_DockerTag": "rhel7_prereqs_2",
-            "PB_AdditionalBuildArguments":"-PortableBuild=true",
+            "PB_AdditionalBuildArguments":"-PortableBuild=true -strip-symbols",
```

Comment:
Why is `-strip-symbols` opt-in?  Shouldn't it be the default since all of our builds opt-in to it?